docs/pages/cdp/cdp-Requirements-on-zOS.md

```diff
@@ -26,3 +26,7 @@ The [DFHDPLOY](https://www.ibm.com/support/knowledgecenter/SSGMCP_5.5.0/applicat
 ### CICSPlex System Manager
 
 CICSPlex System Manager \(CPSM\) is provided with CICS and is required to run the DFHDPLOY utility, and for the [`zowe cics-deploy push bundle`](cdp-CLIReadMe#push--p) command to query application resources via the CICS Management Client Interface (CMCI). CPSM should be connected to the CICS regions into which the application is being installed.
+
+### Node.js SDK
```

Comment:
Suggest changing to IBM SDK for Node.js - z/OS as "Node.js SDK" is neither a product nor a general term (unlike Java SDK).